ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-rhacs-docs-4.5.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: openshift-docs-asciidoc
+    namespace: ocp
+    tag: latest
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 200m
+      memory: 400Mi
+tests:
+- as: deploy-preview
+  steps:
+    env:
+      DISTROS: openshift-acs
+      PREVIEW_SITE: ocpdocs-pr
+    test:
+    - ref: openshift-docs-build-docs
+    - ref: openshift-docs-preview-comment-pages
+- as: validate-asciidoc
+  steps:
+    env:
+      BUILD: build.py
```

Comment:
This should be `BUILD: build_for_portal.py`. I ma updating the generate script now.